flipperdevices/flipperzero-firmware
# [FL-3914] BackUSB

# What's new

- Revert #3931
- Update UI to make switching between modes easier

# Verification 

- Try using the app in both modes, check that everything works as expected even when switching modes
- Check that changing the keyboard layout and removing BLE pairing work

# Checklist (For Reviewer)
